### PR TITLE
What if, Mat had a different look

### PR DESCRIPTION
**Pull request by The1Penguin**
The way I produced the css... is not something that I am proud of

### CODE REVIEW

File: static/style.css
Comment (malmz) [inline static/style.css:1]: Oh no...

**Review by SwedishSubmarine — COMMENTED** [review]
Finally mat.dtek is a website that looks *not bad*!

It also finally respects my system font, so now i can send u3836 screenshots of all my l's on mat.dtek and watch him get upset. 

Looks really good right now but didn't have time to look when there was actually food on the website so will check again on monday and approve then.

**Comment by The1Penguin** [comment]
@SwedishSubmarine If you check now, it will show tomorrows menu

**Comment by SwedishSubmarine** [comment]
I have two issues now that I'm looking on more devices and with the food. 

1. Font. 
The website looks great when it uses a system font I like, but the standard font that my MacBook and iPhone use does not look very good on this website and we should maybe just have a standard font instead. The one mat currently uses is ok i.m.o but I'm curious how it looks with the new style. 

2. Spacing and wrapping.
While the website looks good in fullscreen, it looks quite strange when the window is tiled. Compare the two pictures I'm including, the first is this PR and the second is current. 
bild
bild
I don't necessarily have a problem with the new look, but the large empty space under L's kitchen and S.M.A.K don't look very good to me.

**Comment by The1Penguin** [comment]
@SwedishSubmarine Now uses a better grid for smaller devices and uses sans-serif by default

**Review by malmz — APPROVED** [review]
Does not look sus, good shit!

**Review by SwedishSubmarine — COMMENTED** [review]
Are we supposed to have a set font or is it supposed to be overwritten by system font?

**Comment by malmz** [comment]
> Are we supposed to have a set font or is it supposed to be overwritten by system font?

font-sans sets a resonable sans-serif font from platform system fonts

**Comment by rachelambda** [comment]
I personally way prefer the old serif font, the switch to sans serif feels a bit yucky.

**Comment by SwedishSubmarine** [comment]
Yea I'm of the same mind, I'm not a big fan of sans-serif and especially not the one MacOS and iOS picks by default. Which I do think matters considering how popular these are.

To be clear I won't block here, I do not want to dictate design of this website all on my own, I am okay with it looking like it does now, I just want to get my thoughts written down I suppose. It's definitely more neutral and it's not ugly at all.

**Comment by rachelambda** [comment]
I *do* think that the new design looks good when I switch to a serif font for what it's worth :)

image

(not that I am suggesting this to be the used font)

**Comment by The1Penguin** [comment]
Configure your browsers correctly then ;P

**Comment by rachelambda** [comment]
configuring my browser to using a serif font for sans serif is configuring it correctly?

**Comment by The1Penguin** [comment]
> configuring my browser to using a serif font for sans serif is configuring it correctly?

yes

**Comment by SwedishSubmarine** [comment]
Surely webkit on iOS allows you to customize fonts

**Comment by SwedishSubmarine** [comment]
I'm going to leave this here. The design is good and I have no issues with it, I simply dislike sans serif lol 
:+1: very swag

**Comment by rachelambda** [comment]
awawawa

**Comment by SKyletoft** [comment]
EW EW EW EW

**Comment by SKyletoft** [comment]
VARFÖR HATAR DU VÅRA STACKARS ÖGON?!

**Pull request merged**